### ISSUE TITLE
Model Load Error

### ISSUE THREAD

**Issue by ZanebonoAlter** [w-okada/voice-changer#178]
Envirment: Ubuntu 22.04.2
Python: 3.10.10
My start_command:
python3 MMVCServerSIO.py -p 18888 --https true --modelType so-vits-svc-40 --hubert ./checkpoint_best_legacy_500.pt

error happens when I upload so-vits-model to GUI
[Voice Changer] Model Load Error! Check your model is valid. SynthesizerTrn.__init__() missing 17 required positional arguments: 'segment_size', 'inter_channels', 'hidden_channels', 'filter_channels', 'n_heads', 'n_layers', 'kernel_size', 'p_dropout', 'resblock', 'resblock_kernel_sizes', 'resblock_dilation_sizes', 'upsample_rates', 'upsample_initial_channel', 'upsample_kernel_sizes', 'gin_channels', 'ssl_dim', and 'n_speakers'

this is my config.json

{
  "train": {
    "log_interval": 200,
    "eval_interval": 800,
    "seed": 1234,
    "epochs": 8000,
    "learning_rate": 0.0001,
    "betas": [
      0.8,
      0.99
    ],
    "eps": 1e-09,
    "batch_size": 15,
    "fp16_run": false,
    "lr_decay": 0.999875,
    "segment_size": 10240,
    "init_lr_ratio": 1,
    "warmup_epochs": 0,
    "c_mel": 45,
    "c_kl": 1.0,
    "use_sr": true,
    "max_speclen": 512,
    "port": "8001",
    "keep_ckpts": 3,
    "all_in_mem": true
  },
  "data": {
    "training_files": "filelists/train.txt",
    "validation_files": "filelists/val.txt",
    "max_wav_value": 32768.0,
    "sampling_rate": 44100,
    "filter_length": 2048,
    "hop_length": 512,
    "win_length": 2048,
    "n_mel_channels": 80,
    "mel_fmin": 0.0,
    "mel_fmax": 22050
  },
  "model": {
    "inter_channels": 192,
    "hidden_channels": 192,
    "filter_channels": 768,
    "n_heads": 2,
    "n_layers": 6,
    "kernel_size": 3,
    "p_dropout": 0.1,
    "resblock": "1",
    "resblock_kernel_sizes": [
      3,
      7,
      11
    ],
    "resblock_dilation_sizes": [
      [
        1,
        3,
        5
      ],
      [
        1,
        3,
        5
      ],
      [
        1,
        3,
        5
      ]
    ],
    "upsample_rates": [
      8,
      8,
      2,
      2,
      2
    ],
    "upsample_initial_channel": 512,
    "upsample_kernel_sizes": [
      16,
      16,
      4,
      4,
      4
    ],
    "n_layers_q": 3,
    "use_spectral_norm": false,
    "gin_channels": 256,
    "ssl_dim": 256,
    "n_speakers": 1
  },
  "spk": {
    "meilina": 0
  }
}

But it works when I use it in Windows with this command
MMVCServerSIO.exe --https off -p 18888 --modelType so-vits-svc-40  --hubert checkpoint_best_legacy_500.pt

**Comment by w-okada** [comment]
Sorry, but Windows version is from the same source as the Linux version, so I can't imagine why the behavior would be different. Please check again to make sure you are uploading the same model and config files.

**Comment by w-okada** [comment]
no res close.
